### DISCUSSION TITLE
Possible Bug on mobile app

### DISCUSSION THREAD

**Discussion by Somedex** [community/community#52872] [Other Feature Feedback, Questions, & Ideas] [answered]
When creating a new list and adding several previously starred repos to it, I get a 500 error and can't see my lists or starred whatsoever. It's as if I have nothing starred. This only occurred once all of my starred items had a list they were organized under, so all items were on a list. Not sure if correlation or not. Tried removing a list item to no avail.

**Comment by Somedex** [comment, marked as answer]
> Now try

I don't see the bug any more. Thank you for following up.

**Comment by Faidzandya** [comment]
- ![Screenshot_20230528-020052.png](https://github.com/community/community/assets/122488619/58035032-7e11-48a1-90d0-b08a58bd1a91)

@rileybroughten []()****__****

**Comment by Faidzandya** [comment]
- ![Screenshot_20230528-020052.png](https://github.com/community/community/assets/122488619/58035032-7e11-48a1-90d0-b08a58bd1a91)

@rileybroughten []()****__****

**Reply by Sapibeltv** [reply]
![1696110437217.jpg](https://github.com/community/community/assets/107956561/15c6c570-8a51-4d2c-b48b-f0b8949e569d)

**Reply by Sapibeltv** [reply]
> - ![Screenshot_20230528-020052.png](https://github.com/community/community/assets/122488619/58035032-7e11-48a1-90d0-b08a58bd1a91)
> 
> @rileybroughten []()****__****

We just got there for a few minutes 😊👍 but Leaving entertainment for you https://github.com/community/community/assets/107956561/5dcc4e17-0b76-49ae-b081-fac725b1a6a0

**Reply by Sapibeltv** [reply]
> ![1696110437217.jpg](https://github.com/community/community/assets/107956561/15c6c570-8a51-4d2c-b48b-f0b8949e569d)
> 
> 

# How to make sure you getting out of your bowring right now ```


[]() https://www.youtube.com/@Sapibeltv 
```

**Reply by Shahid2687** [reply]
Thanks

**Reply by Sapibeltv** [reply]
Sooner than later you can see the filmmaker and with the community of the world anmeus to get back to work on our Sovereignty LORD of GOD FATHER WITHIN ![c54ef2430f3fb85df69ae95c6fcd7e3b.jpg](https://github.com/community/community/assets/107956561/e542d3ff-6247-45af-af23-76c80b3af190)

Keeping My devices updated even if I don't use it

**Comment by Faidzandya** [comment]
- ![Screenshot_20230528-020052.png](https://github.com/community/community/assets/122488619/58035032-7e11-48a1-90d0-b08a58bd1a91)

@rileybroughten []()****__****

**Comment by smartdelover22** [comment]
Good

**Comment by elliotthoward** [comment]
> No try

TTbeT7aEVngtCCKapWNmikw35idS4pbTvB

**Comment by davyus321311** [comment]
[]() @Somedex

**Comment by a4men22** [comment]
> When creating a new list and adding several previously starred repos to it, I get a 500 error and can't see my lists or starred whatsoever. It's as if I have nothing starred. This only occurred once all of my starred items had a list they were organized under, so all items were on a list. Not sure if correlation or not. Tried removing a list item to no avail. 
>

**Comment by Sapibeltv** [comment]
> > When creating a new list and adding several previously starred repos to it, I get a 500 error and can't see my lists or starred whatsoever. It's as if I have nothing starred. This only occurred once all of my starred items had a list they were organized under, so all items were on a list. Not sure if correlation or not. Tried removing a list item to no avail. 
> > 
> 
> 

Keep it up until ended up to next level 🤠🤠🤠 before gave us a space to get https://github.com/community/community/assets/107956561/b39468e1-2d5d-4351-a921-1b1fdf8bf41c

on

**Comment by eenthropy** [comment]
> > Now try
> karzon@microsoft.com.
> I don't see the bug any more. Thank you for following up. 

karzon@microsoft.cz

**Reply by eenthropy** [reply]
#

**Reply by eenthropy** [reply]
# - [ ] ## ~~~~

**Reply by eenthropy** [reply]
> # 

# - [ ] # - [ ] ## ~~~~

**Reply by eenthropy** [reply]
- [ ] - @Traveler82 @karzone@eenthropy

**Reply by a4men22** [reply]
![README.md](https://github.com/community/community/files/12820506/README.md)

**Reply by a4men22** [reply]
> > # 
> 
> # - [ ] # - [ ] ## ~~~~

@a4men22 ![release-notes.md](https://github.com/community/community/files/12820525/release-notes.md)

**Reply by a4men22** [reply]
![README.md](https://github.com/community/community/files/12820545/README.md)

**Reply by a4men22** [reply]
> > > Now try
> > karzon@microsoft.com.
> > I don't see the bug any more. Thank you for following up. 
> 
> karzon@microsoft.cz

**Comment by eenthropy** [comment]
> > Now try
> 
> I don't see the bug any more. Thank you for following up.

**Comment by eenthropy** [comment]
# ~~~~

**Comment by Nysory0** [comment]
0886097324

**Reply by Nysory0** [reply]
Phone number

**Comment by mrifix-bir** [comment]
Hello brother tray to find new way  adb code to eneble  divice without   setting

**Comment by hothanhloc68** [comment]
"pm grant by4a.setedit22 android.permission.WRITE_SECURE_SETTINGS".

**Comment by hankqqu** [comment]
> When creating a new list and adding several previously starred repos to it, I get a 500 error and can't see my lists or starred whatsoever. It's as if I have nothing starred. This only occurred once all of my starred items had a list they were organized under, so all items were on a list. Not sure if correlation or not. Tried removing a list item to no avail. 
>

**Reply by hankqqu** [reply]
When creating a new list and adding several previously starred repos to it, I get a 500 error and can't see my lists or starred whatsoever. It's as if I have nothing starred. This only occurred once all of my starred items had a list they were organized under, so all items were on a list. Not sure if correlation or not. Tried removing a list item to no avail.

**Reply by hankqqu** [reply]
> > When creating a new list and adding several previously starred repos to it, I get a 500 error and can't see my lists or starred whatsoever. It's as if I have nothing starred. This only occurred once all of my starred items had a list they were organized under, so all items were on a list. Not sure if correlation or not. Tried removing a list item to no avail. 
> > 
> 
>

**Reply by hankqqu** [reply]
- [ ]

**Comment by hothanhloc68** [comment]
> When creating a new list and adding several previously starred repos to it, I get a 500 error and can't see my lists or starred whatsoever. It's as if I have nothing starred. This only occurred once all of my starred items had a list they were organized under, so all items were on a list. Not sure if correlation or not. Tried removing a list item to no avail. 
> 
"pm grant by4a.setedit22 android.permission.WRITE_SECURE_SETTINGS".

**Comment by hothanhloc68** [comment]
[]()"pm grant by4a.setedit22 android.permission.WRITE_SECURE_SETTINGS".

**Comment by hankqqu** [comment]
> - ![Screenshot_20230528-020052.png](https://github.com/community/community/assets/122488619/58035032-7e11-48a1-90d0-b08a58bd1a91)
> 
> @rileybroughten []()****__****
